pkg/cassandra/config/config.go

```diff
@@ -58,6 +58,24 @@ type Schema struct {
 	// while connecting to the Cassandra Cluster. This is useful for connecting to clusters, like Azure Cosmos DB,
 	// that do not support SnappyCompression.
 	DisableCompression bool `mapstructure:"disable_compression"`
+	//Datacenter is the name for network topology
+	Datacenter string `mapstructure:"datacenter" valid:"optional"`
+	// TraceTTL is Time To Live (TTL) for the trace data in seconds
+	TraceTTL int `mapstructure:"trace_ttl" valid:"optional"`
+	// DependenciesTTL is Time To Live (TTL) for dependencies data in seconds
+	DependenciesTTL int `mapstructure:"dependencies_ttl" valid:"optional"`
+	// Replication factor for the db
+	ReplicationFactor int `mapstructure:"replication_factor" valid:"optional"`
+	// CasVersion is version of cassandra used
+	CasVersion int `mapstructure:"cas_version" valid:"optional"`
+	// CompactionWindow of format "^[0-9]+[mhd]$" tells the compaction window of the db
+	CompactionWindow string `mapstructure:"compaction_window" valid:"optional"`
```

Comment:
so this is defining a time interval? Can we then also use time.Duration type?

Reply:
Only time precision form https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/TimeUnit.html are allowed as per https://cassandra.apache.org/doc/latest/cassandra/managing/operating/compaction/twcs.html.

Also, below is based on the current script used. Should we go ahead with breaking it?
`// CompactionWindow of format "^[0-9]+[mhd]$" tells the compaction window of the db`

Reply:
we are creating new config, not bound by previous restrictions. If the destination API does not allow smaller units we can always add validation `cw >= time.Minute`

Reply:
Updated along with a validator